code/game/objects/items/weapons/autolathe_disks.dm

```diff
@@ -1188,22 +1188,75 @@
 	)
 
 /obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee
-	disk_name = "NeoTheology Armory - Melee Weapons"
+	disk_name = "NeoTheology Armory - Basic Melee Weapons"
 	icon_state = "neotheology"
 	spawn_blacklisted = TRUE
 	rarity_value = 50
 	license = -1
 	designs = list(
 		/datum/design/autolathe/sword/nt_sword,
-		/datum/design/autolathe/sword/nt_longsword,
 		/datum/design/autolathe/sword/nt_dagger,
-		/datum/design/autolathe/sword/nt_halberd,
+		/datum/design/bioprinter/storage/sheath,
+		/datum/design/autolathe/tool_upgrade/sanctifier
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee/longsword
+	disk_name = "NeoTheology Armory - NT Longsword"
+	icon_state = "neotheology"
+	spawn_blacklisted = TRUE
+	rarity_value = 50
+	license = -1
+	designs = list(
+		/datum/design/autolathe/sword/nt_longsword,
+		/datum/design/bioprinter/storage/sheath,
+		/datum/design/autolathe/tool_upgrade/sanctifier
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee/scourge
+	disk_name = "NeoTheology Armory - NT Scourge"
+	icon_state = "neotheology"
+	spawn_blacklisted = TRUE
+	rarity_value = 50
+	license = -1
+	designs = list(
 		/datum/design/autolathe/sword/nt_scourge,
+		/datum/design/bioprinter/storage/sheath,
+		/datum/design/autolathe/tool_upgrade/sanctifier
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee/halberd
+	disk_name = "NeoTheology Armory - NT Halberd"
+	icon_state = "neotheology"
+	spawn_blacklisted = TRUE
+	rarity_value = 50
+	license = -1
+	designs = list(
+		/datum/design/autolathe/sword/nt_halberd,
+		/datum/design/bioprinter/storage/sheath,
+		/datum/design/autolathe/tool_upgrade/sanctifier
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee/shield
+	disk_name = "NeoTheology Armory - NT Shield"
+	icon_state = "neotheology"
+	spawn_blacklisted = TRUE
+	rarity_value = 50
+	license = -1
+	designs = list(
 		/datum/design/autolathe/shield/nt_shield,
 		/datum/design/bioprinter/storage/sheath,
```

Comment:
Why include sheath and sactifier in the shield design disk?

Reply:
Just so that disk don't feel empty.

Reply:
You know how empty feels disk with only one design.
He need a friend.

Reply:
Go ahead and make it contain designs for the shortsword and sheath as well as the shield

Reply:
Sure.

Reply:
Done